orchagent/orch.cpp

```diff
@@ -242,14 +244,16 @@ void Consumer::execute()
     // ConsumerBase::execute_impl<swss::ConsumerTableBase>();
     SWSS_LOG_ENTER();
 
-    size_t update_size = 0;
     auto table = static_cast<swss::ConsumerTableBase *>(getSelectable());
+    int batch_count = 0;
+    size_t update_size = 0;
```

Comment:
Will move this loop to swss-common ConsumerStateTable::pops() method.

https://github.com/sonic-net/sonic-swss-common/pull/916